.github/workflows/lint-test.yml

```diff
@@ -22,6 +22,8 @@ jobs:
     env:
       API_URL: 'ws://172.16.238.10:9944'
       TEST_PALLET_ADDRESS: ${{ secrets.TEST_PALLET_ADDRESS }}
+      XYK_PALLET_ADDRESS: ${{ secrets.XYK_PALLET_ADDRESS }}
```

Comment:
@EddyJLSH , I added a prefix "E2E_" in the secrets, to be consistent with node-repo. do you mind update those names?
We can leave the Environment variables they are, but secrets are:
E2E_XYK_PALLET_ADDRESS
and
E2E_TREASURY_PALLET_ADDRESS
Thanks!